test/helpers/sendTransaction.js

```diff
@@ -15,7 +15,15 @@ function sendTransaction (target, name, argsTypes, argsValues, opts) {
   return target.sendTransaction(Object.assign({ data: encodedData }, opts));
 }
 
+function sendEther (from, to, value) {
+  web3.eth.sendTransaction({
+    from: from,
+    to: to,
+    value: value,
```

Comment:
Let's add `gasPrice: 0` here, since that will only cause issues when calculating balance differences.

Reply:
will push in next commit. Thanks